lib/sync/sync-metrics.js

```diff
@@ -191,7 +192,8 @@ module.exports = {
       metricsConfig.backends.push({
         type: 'influxdb',
         host: syncConfig.metricsInfluxdbHost,
-        port: syncConfig.metricsInfluxdbPort
+        port: syncConfig.metricsInfluxdbPort,
+        sendQConcurrency: syncConfig.metricsReportConcurrency
```

Comment:
Is this option not `sendQueueConcurrency`?  I think `sendQConcurrency` is only used internally.

https://github.com/feedhenry/fh-component-metrics/blob/b8565be0205aed2f8eafe3cfd8be615db5c3d679/lib/clients/base.js#L9